Lab78.ipynb

```diff
@@ -0,0 +1,575 @@
YC+BuleeLN9Wzgzt8L2oqqwxrNxXP1lrozF+KUpdQI2qe+GVOHObpJazasCVv19c5rqgZdwTHpU9XD2/3RuhMmLsywzRTWWU46YExqWW3uMWagjIbL4cwFRWLbVUmbXrPmpItz5Oi1BdUKbhYtm4zt77xdcZoWbAG1dz1znTfkZBem/bet/wvb+enu74/57FxrLQrcLciCBNl4w3ZrG1/ld8I36ioqjJsKWCmMkVRglGl4OL/XpvKI6O/y0i+5VTMf3t/Zq1FBTjjBNxnG2/H62djrWeE7PoiT4QSJ4rdU1CUJBE0JqSmqFJw4Tg1g2aAArgzR7bNYuFE/bht5mGSn3nNW0GjT+sC81ZsUKVQAw7q3q7UIig1wDsnRLFQpeCDE9Hz1Kdzmf79WqZ9X52fpRgRNWGoNKYgG3e2id+PuffjmogUOy55akJGBFWS2X2nlrV6vhP7darV8ynFJaoBbbGJPhKRXsAvgXbAe8aYv9fGeQ/64/uc3K8TVx3ZM8P+HmbYe1RUVhq2Sf6t4GKnEAhDs0blOjK3APp33Y49OrbiSTsrZ4fWTYo25iIMhc0rpsSFqJIpRtpTEJHHRGSJiEzxrD9aRKaLyEwRGQFgjJlmjLkUOA0YEKVcbhas3Mi976fn2o/Dx7KtqgopQJIv5uX2OxSbwxOS1M3Nuft3LbUIzF2+Ia21vl3zRln2Lg33nLF3Qcf95pjdiyxJOs9dtF+k5SeB4/fqGEm5UZuPHgeOdq8QkXLgfmAo0BsYLiK97W3HA6OB4CxSRcQ7WMmZBenipyakJkopFVXG5Mxr5Ee4yUuKy2kDOvO34f1o6JM2Oa7cfMKekZY/oOt2OfdZtm5zutrPEcNw2WG71kimfHj8/H159bIDOWHvTrxxxUEZ24fuuVPW43+0+w5Ztzs0a1TYKPK8cnLVUVo1aRhJuZEqBWPMKMCbHWwgMNMYM9sYswV4HjjB3v81Y8wBwJlBZYrIxSIyXkTGL11aM+97tsicYs9olS9D7h7FoCwZFuNE4wblDNurY8ETnddFDgtZKeZzz7Zv3rhQcXyVVLZTH9pzB/bauQ3gXwH//az+Wc/XJMtUpGkyhNqrmnuH9+OpCwcGZu9NAk0aFqfajepzK4WjuRMw37W8AOgkIoeKyL0i8hDwZtDBxpiHjTEDjDED2rdvH7RbKEphB9+hZeEfdlxpa5s98p3UJkmctZ+VLqRi+2ah9veb2tPLpYfUvOU/sKJtqP38KvawlUqYa/GSbX7qdBnyK/uQ3dozuEd73+OO69sh8LgdW0X33Z29X36myBP2ynTwX35Y/sk9g/KS1ZRSKAW/KzHGmA+NMb8wxlxijLk/awEiw0Tk4dWrs090notPZ9e+iWiHCF/OUtCovIzuO7RIW3fUHqXzMey2Ywv+87MDcu8INM/DdHHFj3oAcOHgXULt3yBExtWgb7pv59Z8ev3hofcvlLA+q1ymmhcv3T9t+ekLB9E05L3N95Kce+CVaeINR/CbY3oFHvd/w/JPRRMWv9nmsuE31mlI7/y/mahMaKVQCguAnV3LnYHcs8O4MMa8boy5uHXr1jUSxJ1/6JbX85tIvdDnUV5Wt6KAB/fIjHUPazqIggZlZfR3mUqy9cxa5mGTbdu8EXPuOJbj+4Zz7oWxlZcJ9OqQGYa6Q8vG7NQ6MxeWb2vKdkQ8dHb/rOf0VkPts9yXrp7ekHfeZS/7VrRN8ycZDE0ahO0pBG9r2aQB//lZusJxdvd+f2USXElu37xRrHwQfq6jQnoydcl89DnQQ0S6iUgj4AzgtXwKKFZPwc1jIedZddixlX8Cu1wU0hWPM34vZnmeb+t7Vx9SJGmgoWdO3t07tArcN584fcn4kcnEG45I/W7ROHe0d5kIjRuUZzhtgwbN+5lMnH3LRLKbE1xlPnR2f0b+ItN57PDnU/dKlzNELZE2DauBspDveTbzUZlIxhwXzjV6r1WQwEpSJP93Miz5mo7ACnAZMTQ9OqsQ+aK6pqhDUp8DxgI9RWSBiFxojNkGXA68DUwDXjDGTM1Wjpdi9RRqQq6pHb388nDL/FCsB/m7Y4O7yrWLz/XkcYndd2hRVNtowzyU7rVH9eTZiwZx92nVlaDf4beeuGfKHJSt+LaukFLvhPHNGpVzyG6Zqc8hU7EGBSFlu01C+Ns+oOt27NCySWB53oZLrp4CpJtEnF9z7jg2ZxSWCHTerqnvtvIyyTBxOTJnKIWybN+WUJ7FxNOuReGhwD12rDaddmrjfx1eqoyhddP0XmohQRqJ9CkYY4YbYzoYYxoaYzobYx61179pjNnNGLOrMea2fMuNoqeQL/26tMlrf6f7mm0qvyTw8s8PSBt561tJhkwP9eE1h/LSzw/Iy67cpll2k4+3hZqt7LIy4YBd26WNs/D70M5ytQaDPt6Hzk6PxslsyVr+jrR19j7eii9ovupst7qsLLtJyD1lZdA19O1sNbK8sofpKQT1bi44KH2GwV/YjSOHU/t35t2rDmHKzUdlHFsmma1/5155w5+FYPNRmWTvoe+3y/aB23LhDq7wXmsQVQb26Jjegy3EvBW2N5Z3uZGUGjGl7imcNqAz+3TJHYfuZrI9x8GX81fl2DMcUbUSvHRp24xT9qmeAKhfl+3SWjm+5qMy4dCe6a1iv/e3ol1zWjVpmJdt9PR9d86+g6ty8soQBvd93W+XthmpJ/xEvfG43hy1h2UCeu3yA7nr1L0y9vOriJ1zOZsc0QtJuSgIT/90UMh9/XEq9gxR8xTIrdSO6dMhrQF11RG7MfZ6a7raMoHrh/aiScNyX3NbmWTK4iy3aNLAs69k3ON3rjw4dUy2Sve6owsfaOdt5B29R/bxG2DJ3rdzmzRzY5B4++0SHF0WlSU6kUqh1GR2anNT7DwltTUkoEnDMrbztM7drUI/5dSoQRmPnz8wbV02JZaPgrvuqN19I3NSstk12Ne3HMUj54QbGJ/WMneJ8sCZ/fnvrw7OKev5B1akfvft3IZ
```

Comment:
Could you make these parts from the lab instructions bold? I didn't notice the answers you had in this section at first, they were blended in with the instructions